chia/plotting/manager.py

```diff
@@ -81,6 +162,24 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._lock.release()
 
+    def cache_path(self):
+        path = self.root_path.resolve() / "cache" / "plot_manager.dat"
+        mkdir(path.parent)
+        return path
+
+    def load_cache(self):
+        try:
+            self.cache.load_from(self.cache_path())
+        except Exception as e:
```

Comment:
I think we should catch `FileNotFoundError` here, and not print anything to the log. Otherwuse, the first time you start you'll get this in the log:

```
09:23:15 chia.plotting.manager: ERROR Failed to load cache: [Errno 2] No such file or directory: '/private/var/folders/s_/_vk8b8gj6x5_d1p2vpr5r5gh0000gn/T/tmpq5lc9wup/cache/plot_manager.dat', Traceback (most recent call last):
  File "/Users/arvid/Documents/dev/chia-blockchain/chia/plotting/manager.py", line 170, in load_cache
    self.cache.load_from(self.cache_path())
  File "/Users/arvid/Documents/dev/chia-blockchain/chia/plotting/manager.py", line 87, in load_from
    serialized = path.read_bytes()
  File "/opt/homebrew/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py", line 1249, in read_bytes
    with self.open(mode='rb') as f:
  File "/opt/homebrew/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py", line 1242, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/opt/homebrew/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pathlib.py", line 1110, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/s_/_vk8b8gj6x5_d1p2vpr5r5gh0000gn/T/tmpq5lc9wup/cache/plot_manager.dat'
```

Reply:
Good one, added!